src/electron/main.js

```diff
@@ -8,18 +8,12 @@ const {
 const { menubar } = require('menubar');
 const { autoUpdater } = require('electron-updater');
 const { onFirstRunMaybe } = require('./first-run');
-const path = require('node:path');
 
 // TODO: Remove @electron/remote use - see #650
 require('@electron/remote/main').initialize();
 
-const idleIcon = path.join(
-  __dirname,
-  'assets',
-  'images',
-  'tray-idleTemplate.png',
-);
-const activeIcon = path.join(__dirname, 'assets', 'images', 'tray-active.png');
+const idleIcon = `${__dirname}/../../assets/images/tray-idleTemplate.png`;
```

Comment:
From what I've read about electron, this may not work in the packaged version

Reply:
Could be a non issue. Worth validating with the dmg

Reply:
Testing now! 🤓

Reply:
It did indeed not work, but I'll make a change to not use join nonetheless

Reply:
Done! Now working with path.resolve, which is way cleaner than join!